lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -4832,7 +4832,7 @@ TypeSystemClang::GetBitSize(lldb::opaque_compiler_type_t type,
       return bit_size;
   }
 
-  return std::nullopt;
+  return llvm::createStringError("could not get size of type");
```

Comment:
Should we include the `TypeClass` in the error message and/or even the typename?